packages.json

```diff
@@ -2197,11 +2197,15 @@
         "url": "https://github.com/jhonny-me/YCode-ios-xcode-template.git",
         "description": "Xcode template to quick start for a commercial development"
       },
-      {
```

Comment:
Looks like you removed this by accident, which broke the JSON, would you mind fixing that?